plugins/base_commands.py

```diff
@@ -8,8 +8,12 @@
 
 import datetime
 
+import logging
 from spockbot.plugins.base import PluginBase
 from spockbot.vector import Vector3
+import json
+tpareqs = {}
```

Comment:
Fair enough, thanks for the tips. Hopefully this can be useful to somebody to some end :)